buildroot-external/package/rpi-eeprom/Config.in

```diff
@@ -1,6 +1,37 @@
 config BR2_PACKAGE_RPI_EEPROM
 	bool "rpi-eeprom"
+	select BR2_PACKAGE_PYTHON3
```

Comment:
I guess this is the biggest contributor to the size increase for this feature?

I assume we have enough free space on Pi's since we don't ship it with that many drivers and firmwares? :thinking:

Reply:
Yep, that's what I wrote in the last paragraph. We still have plenty of space (the bloat is from 170M to ~180M), so there's no big concern we would need to evict it soon. It just feels like an overkill to add it just to parse some binary files, that's all.